src/main/initNrfUtilProxyEnv.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+export default () => {
+    const index = process.argv.findIndex(p => p.includes('--proxy-server'));
```

Comment:
C: In theory, the argument to `--proxy-server` could be quite complicated. E.g. on https://www.chromium.org/developers/design-documents/network-settings/#command-line-options-for-proxy-settings an example is `--proxy-server="http=foopy:80;ftp=foopy2"`. But I think it is ok if we do not support this here until a customer requests it.